### ISSUE TITLE
Google Analytics I have won the hant

### ISSUE THREAD

**Issue by gianarb** [zendframework/modules.zendframework.com#479]
:tada: 
ladies and gentlemen, today I'm here to announce the recovery of google analytics account!

The history begins here #139
The owner of this account is @EvanDotPro and now I have the access but where is the tracking? The graph is empty and this site not embed the google analytics track code..

## 02/01/2015 - Today
![02-genn-2015](https://cloud.githubusercontent.com/assets/1630267/6537597/bf65a37a-c458-11e4-9e26-50eadcbabacb.png)

## Code
```html
<script>
  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
  ga('create', 'UA-39824791-1', 'auto');
  ga('send', 'pageview');
</script>
```

**Comment by localheinz** [comment]
:+1:

**Comment by gianarb** [comment]
@localheinz would you revert the commit that remove the code or send new PR? :)

**Comment by ins0** [comment]
@gianarb done 

/cc @localheinz

**Comment by localheinz** [comment]
Fixed in #480.
